misc/kickstart_create_network_line.py

```diff
@@ -170,8 +170,7 @@ def useable_interfaces(net_devs, nc, iface_speed):
         #  if not it is anyway.
         # Thus we are doing a bond of some sort.
         # This gives us the fastest interfaces first:
-        speeds = net_devs.keys()
-        speeds.sort(reverse=True)
+        speeds = [k for k in sorted(net_devs.keys(), reverse=True)]
```

Comment:
you just couldn't resist

Reply:
That was the Python3 compatibility error! (`dict_keys` has no member `sort`)

Reply:
Oh, I see. I thought you were just making it more pythonic!

Reply:
No, although that was my pleasure.